person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
```

Comment:
- [ ] You need to set up the other end of the has-many-through relationship here in the person class.

  ```
  def add_rental(book)
    Rental.new(self, book)
  end
  ```